servers_v6.json

```
@@ -58,5 +58,8 @@
   {
     "name": "Minty [subzero]",
     "address": ["minty-server.ddns.net"]
+  },
+  {
+    "address": ["shizashizashiza.ml:6567"]
```

6567 port is default, you can omit it
```suggestion
    "address": ["shizashizashiza.ml"]
```